java/openjdk/Portfile

```diff
@@ -273,6 +276,24 @@ subport openjdk11-openj9-large-heap {
     replaced_by  openjdk11-openj9
 }
 
+subport openjdk11-sap {
+    # https://github.com/SAP/SapMachine/releases/tag/
```

Comment:
There is no page at this URL. I suggest using https://github.com/SAP/SapMachine/releases/ instead.